tests/integration/models/device.spec.coffee

```diff
@@ -2087,7 +2087,43 @@ describe 'Device Model', ->
 						target_os_version
 					)).to.throw('The device is offline')
 
-			it 'should throw when the device is running a dev version', ->
+			it 'should throw for upgrades from prod -> dev', ->
+				[
+					['Resin OS 2.0.0+rev3 (prod)', 'prod']
+					['Resin OS 2.0.0+rev3 (prod)', '']
+					['Resin OS 2.0.0+rev5 (prod)', '']
+					['Resin OS 2.0.2+rev2', 'prod']
+					['Resin OS 2.0.3+rev1', 'prod']
+					['Resin OS 2.0.4+rev1', 'prod']
+					['Resin OS 2.0.5', 'prod']
+					['Resin OS 2.0.5+rev1', 'prod']
+					['Resin OS 2.0.6+rev1', 'prod']
+					['Resin OS 2.0.6+rev2', 'prod']
```

Comment:
We can drop some of these as there is a lot of redundancy.

Reply:
Removed some similar cases.